prime-router/src/main/kotlin/azure/WorkflowEngine.kt

```diff
@@ -259,7 +261,7 @@ class WorkflowEngine(
      */
     fun handleReportEvent(
         messageEvent: ReportEvent,
-        updateBlock: (header: Header, retryToken: RetryToken?, txn: Configuration?) -> ReportEvent,
+        updateBlock: (header: Header, retryToken: RetryToken?, txn: Configuration?) -> ReportEvent
```

Comment:
Here and in a lot of other spots, the formatting change is removing the trailing comma. From the [Kotlin coding conventions](https://kotlinlang.org/docs/coding-conventions.html#trailing-commas):

> The Kotlin style guide encourages the use of trailing commas at the declaration site and leaves it at your discretion for the call site.

I personally prefer trailing commas wherever allowed by the language, but it sounds like particularly for function definitions, the standard is to keep them. It would be nice to get to the bottom of what is leading to these formatting changes, whether it is `ktlint` or some other general formatter. Unless there's a project specific standard established somewhere, it seems like we should have settings that match the published Kotlin conventions. This is outside the scope of the issue though, so no need to get too hung up on it right now.

Reply:
I'm not sure when our ktlint was last updated.. I'm gonna check that, might be work making a ticket to refresh it